src/webapp/src/client/app/layout/left-menu/left-menu.controller.js

```diff
@@ -114,5 +130,9 @@
         function isSidebarOpen() {
             return leftMenuService.getStatus();
         }
+
+        function isMenuExpanded(groupName) {
```

Comment:
Is that empty method really needed here?

Reply:
Removed.